src/core/engine.js

```diff
@@ -421,20 +694,100 @@ function engineFactory({ expression = '', position = 0, maths = {} } = {}) {
          * @returns {calculator}
          * @fires commandadd after each command has been registered
          */
-        setCommands(defs) {
+        setCommandList(defs) {
```

Comment:
**thought:** the previous names `getCommands`/`setCommands` seemed more natural than `getAllCommands`/`setCommandList`/`clearCommands`. But if you like it, leave it.

Reply:
Well, the `getCommands`/`setCommands` were ambiguous in the sense it reduced the readability because the names for the multiple and the single were too close. I wanted to give a clearer naming to avoid silly errors.